app/views/layouts/application.html.erb

```diff
@@ -64,7 +64,7 @@
       </div>
     </nav>
     <div
-      class="bg-image"style="background-image: url('https://img.freepik.com/premium-photo/smooth-empty-grey-studio-well-use-as-background_1258-3149.jpg?w=2000');height: 120vh;"> 
+      class="bg-image"style="background-image: url('https://www.purolatorinternational.com/content/uploads/2021/12/iStock-1124533318-1.jpg');height: 100vh;"> 
```

Comment:
shift this styling into css file.

Reply:
rails 7 is problematic. Working on it. Couldn't work with custom css files before